ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -589,6 +595,17 @@ List<URL> getListeners() {
     }).collect(Collectors.toList());
   }
 
+  // Current tests require URIs as URLs, even though they're not URLs
+  List<URL> getProxyProtocolListeners() {
```

Comment:
Is this method used? I don't see it called from anywhere.

Reply:
Done